app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  validates :name, presence: true
+  has_many :posts
+
```

Comment:
- [ ] It would be nice if you make the one-to-many relationship of your user table with comments and likes tables too to follow the ER Diagram provided by microverse in the project requirements 👍
- [ ] Second author_id is the alias of user_id so it would be better if you link author_id instead of user_id in the relationships 😊 After adding the relationship it will look something like this ⬇️

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

```